gstreamer.jhbuildrc

```diff
@@ -12,7 +12,13 @@ build_policy = 'updated-deps'
 autogenargs = ''
 autogenargs += ' --cache-file=' + checkoutroot + '/autoconf-cache'
 
-os.environ['INSTALL'] = os.path.expanduser('~/local/bin/install-check')
+_install_check = os.path.expanduser('~/.local/bin/install-check')
```

Comment:
You pull-requested this file change in #15 as well and this file change is not related to the jhbuild. So I think you included two changes in one PR in accident.